test/bigtest/helpers/turn-off-errors.js

```diff
@@ -0,0 +1,14 @@
+const error = console.error;
+const blacklist = [
+  /Warning: Encountered two children with the same key/,
+  /Warning: Each child in a list should have a unique "key" prop/,
```

Comment:
Are we _really_ confident we want to turn those off? They sound like useful errors to be aware of, though if they are just an artifact of testing, I agree we can hide them.

Reply:
Snapshot produces the exact same warnings when accessing the select service point modal, so it's not an artefact of testing.  I had to turn it off  while writing tests because it gets reported so many times it's difficult to actually read the test results.  I can remove now that it's moving into production.